README.md

```diff
@@ -1,177 +1,164 @@
 
-We need to create a persistent volume claim (storage) to use the storage created in previous step and bind it to the instance of this operator. Click on storage under Web Console and select Persistent Volume Claims.
+1. You need to create a persistent volume claim (storage) to use the storage created in the previous step and bind it to the instance of this operator. On the web console, click on **Storage** and select **Persistent Volume Claims**.
 
-![](https://github.com/IBM/getting-started-with-fppredictplus/blob/master/images/select-pvc.png)
+    ![](https://github.com/IBM/getting-started-with-fppredictplus/blob/master/images/select-pvc.png)
 
-The next step is to create persistent volume claim. We can select Storage Class from Gold, Silver, Bronze which was created in previous step, give the name as `fp-predict-plus-pvc`, select single user access & assign the storage size as 20 GB. 
+1. The next step is to create a persistent volume claim. Choose the storage class you need &mdash; either gold, silver, or bronze. Add your name as `fp-predict-plus-pvc`, select single user access, and assign the storage size as 20 GB. 
 
-![](https://github.com/IBM/getting-started-with-fppredictplus/blob/master/images/create-pvc.png)
+    ![](https://github.com/IBM/getting-started-with-fppredictplus/blob/master/images/create-pvc.png)
 
-After the persistent volume claim (PVC) is created, it needs to be bound with the persistent volume (PV) which was created in earlier step. We should see the status as `Bound` per below after couple of minutes.
+1. After you create the persistent volume claim (PVC), you need to bind it with the persistent volume (PV) which you created in tye earlier step. You should see the status as `Bound` per below after couple of minutes.
 
-![](https://github.com/IBM/getting-started-with-fppredictplus/blob/master/images/pvc-bound.png)
+    ![](https://github.com/IBM/getting-started-with-fppredictplus/blob/master/images/pvc-bound.png)
 
-### Install the operand (Instance) of FPPredict Plus
+## Install the instance of FPPredict Plus
 
 Click on Installed operators under `Operators` and click on FP Predict Plus Operator to get the options like Overview, YAML, Subscription, Events, FP-Predict-Plus. Click on YAML and update the name per below under persistent volume with `useExisting as false`, name of persistent volume claims, routerCanonicalHostname would be the web console URL and hit `Save`. routerCanonicalHostname would start with the cluster name, cluster id till appdomain.cloud. The initial part in the URL - `https://console-openshift-console` should be removed while updating routerCanonicalHostname.
 
-![](https://github.com/IBM/getting-started-with-fppredictplus/blob/master/images/yaml-changes.png)
+![](https://github.com/IBM/getting-started-with-fppredictplus/blob/master/images/yaml-changes.png) <!--EM: Is this the wrong image? It doesnt' seem to go with teh text above-->
 
```

Comment:
The image is correct. This is how the YAML file looks like.

Reply:
I know what YAML looks like. I was confused by the preceding sentence where you're clicking on "Istalled operators". I just made it a separate step.

Reply:
We can't assume all readers can see the YAML image there. Does this work? 

I think we should explicitly write out what areas the reader is supposed to fill out in that YAML file. Can you write it out in a bulleted list?